nibabies/workflows/anatomical/segmentation.py

```diff
@@ -184,6 +185,57 @@ def init_anat_segmentations_wf(
         (split_seg, outputnode, [('out', 'anat_tpms')]),
     ])
     # fmt: on
+
+    wf.__desc__ += """Segmentation volume label indices were remapped
+                     from FreeSurfer to M-CRIB-compatible format. """
+
+    # dictionary to map labels from aseg to mcrib
+    aseg2mcrib = {
+        2: 51,
+        3: 21,
+        4: 49,
+        5: 0,
+        7: 17,
+        8: 17,
+        10: 43,
+        11: 41,
+        12: 47,
+        13: 47,
+        14: 0,
+        15: 0,
+        16: 19,
+        17: 1,
+        18: 3,
+        26: 41,
+        28: 45,
+        31: 49,
+        41: 52,
+        42: 20,
+        43: 50,
+        44: 0,
+        46: 18,
+        47: 18,
+        49: 42,
+        50: 40,
+        51: 46,
+        52: 46,
+        53: 2,
+        54: 4,
+        58: 40,
+        60: 44,
+        63: 50,
+        253: 48,
+    }
+
+    map_labels = pe.Node(MapLabels(mappings=aseg2mcrib), name="map_labels")
+
+    # fmt:off
+    wf.connect([
```

Comment:
Since this is always being run, we can just move all these definitions into the one above and remove this extra connect